### ISSUE TITLE
begin callback not called

### ISSUE THREAD

**Issue by luglio7** [juliangarnier/anime#98]
the begin callback is never triggered, why ?

```
anime({
      targets     : menuLines,
      opacity     : [0,1],
      delay       : (el, i) => i * 200,
      easing      : 'easeOutExpo',
      duration    : 500,
      begin       : () => {
        console.log('begin');
      },
      complete    : () => {
        console.log('complete');
      }
    });
```

**Comment by juliangarnier** [comment]
Looks like a bug, but should be fixed in 1.1.3, can you try with this version https://github.com/juliangarnier/anime/blob/v1.1.3/anime.js ?

Thanks

**Comment by blakewatson** [comment]
I had a problem firing a begin inside of a begin, but v1.1.3 fixed it.
